src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { Navbar } from "../../components";
+import { useAuth } from "../../context";
+
+const Login = () => {
+  const [email, setEmail] = useState("foody@gmail.com");
+  const [password, setPassword] = useState("ilovefood");
```

Comment:
here you can create a single object which has email and password as keys and some value